recipes/sdl/all/conanfile.py

```diff
@@ -184,11 +184,11 @@ def build_requirements(self):
             # set. This could be because you are using a Mac OS X version less than 10.5
             # or because CMake's platform configuration is corrupt.
             # FIXME: Remove once CMake on macOS/M1 CI runners is upgraded.
-            self.tool_requires("cmake/3.25.3")
+            self.tool_requires("cmake/3.27.5")
```

Comment:
```suggestion
            self.tool_requires("cmake/[>3.5 <4]")
```
https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges